nesta/core/orms/arxiv_orm.py

```diff
@@ -81,6 +81,7 @@ class Article(Base):
     institutes = relationship('ArticleInstitute')
     corex_topics = relationship('CorExTopic',
                                 secondary='arxiv_article_corex_topics')
+    article_source = Column(VARCHAR(7), index=True, default=None)
```

Comment:
Note, I opted for an extra column over a look-up, on the basis that there will be very few values of `article_source` so the subsetting time will be tiny. Also, `VARCHAR(7)` will only add a total of 15MB to the database, which is peanuts.